lib/Template/gramma.peg

```diff
@@ -70,12 +70,12 @@ ValuelessAttribute = name:AttributeName {
     };
 }
 
-AttributeName = [a-zA-Z0-9\-]+ { return text() }
+AttributeName = '#'? [a-zA-Z0-9\-]+ { return text() }
```

Comment:
Maybe we can use something like `pragma:`, `#` might be confusing and strange looking. Please consider this as an option